bc_gym_planning_env/envs/synth_turn_env.py

```diff
@@ -233,7 +233,7 @@ def __init__(self, params=None, draw_new_turn_on_reset=True, seed=None, rng=None
         :param rng np.RandomState: random number generator
         """
         if rng is None:
-            self._rng = np.random.RandomState()
+            self._rng = np.random.RandomState(1001)
```

Comment:
Please remove and seed as needed in the parts that rely on repeatable RNG state sequence ie. testing.

```env.seed(1001)``` instead of giving same seed to everyone

Reply:
good call, fixed